frontend/src/App.js

```diff
@@ -1,25 +1,38 @@
-import logo from './logo.svg';
-import './App.css';
+import { HashRouter, Routes, Route } from "react-router-dom";
+import Footer from "./components/Footer";
 
-function App() {
+import Header from "./components/Header";
+import Homepage from "./pages/Homepage";
+import Login from "./pages/Login";
+import Register from "./pages/Register";
+import PageNotFound from "./pages/PageNotFound";
+const App = () => {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="App ui component">
+      <HashRouter>
+        <Header />
+        <Routes>
+          <Route
```

Comment:
create a file `routes.js` and import here instead of directly putting in `app.js`

Reply:
Should i put it on the same level as App.js or put it on a specific folder?